geotrek/common/management/commands/import.py

```diff
@@ -12,7 +12,7 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument('parser', help='Parser class name in var/conf/parsers.py (or dotted syntax in python path)')
-        parser.add_argument('shapefile', nargs="?")
+        parser.add_argument('filename', nargs="?", help='Optional file which will be use to feed database')
```

Comment:
```suggestion
        parser.add_argument('filename', nargs="?", help='Optional file used to feed database')
```
plus court, c'est mieux